openwisp_utils/staticfiles.py

```diff
@@ -25,4 +25,3 @@ def __init__(self, app_names=None, *args, **kwargs):
             filesystem_storage = FileSystemStorage(location=root)
             filesystem_storage.prefix = prefix
             self.storages[root] = filesystem_storage
-        super(FileSystemFinder, self).__init__(*args, **kwargs)
```

Comment:
I deleted this line because https://github.com/django/django/blob/master/django/contrib/staticfiles/finders.py#L52 is overraiding `self.locations` and `self.storages`